app/models/tenant.rb

```diff
@@ -1,6 +1,8 @@
 class Tenant < ApplicationRecord
   scope :current, ->(user) { where(:external_tenant => user.tenant) }
 
+  has_one :order_template
```

Comment:
Technically I want to start with this connecting to a Product (or source) but we currently do not store any object for that in our db.  Might need to consider that as a pre-req.